plugins/inputs/tail/tail.go

```diff
@@ -236,6 +236,10 @@ func (t *Tail) tailNewFiles(fromBeginning bool) error {
 
 				if err := tailer.Err(); err != nil {
 					t.Log.Errorf("Tailing %q: %s", tailer.Filename, err.Error())
+					if strings.HasSuffix(err.Error(), "permission denied") {
+						t.Log.Errorf("Deleting tailer for %q", tailer.Filename)
```

Comment:
How about adding some details like
```suggestion
						t.Log.Errorf("Deleting tailer for %q: %v", tailer.Filename, err)
```

Reply:
I moved the previous log message into an else at the same time, hopefully this is adequately idiomatic.